src/main/java/factionsplusplus/data/beans/PollVoteBean.java

```diff
@@ -0,0 +1,18 @@
+package factionsplusplus.data.beans;
+
+import java.util.UUID;
+
+import org.jdbi.v3.core.mapper.reflect.ColumnName;
+
+import lombok.Data;
+
+@Data
+public class PollVoteBean {
+    private UUID id;
+    @ColumnName("player_id")
+    private UUID playerUUID;
+    @ColumnName("poll_id")
+    private UUID pollUUID;
```

Comment:
This column isn't needed because option_id relates to poll_id already, so poll can be grabbed from option